checkpointing/tensorflow/checkpoint.sub

```diff
@@ -0,0 +1,30 @@
+
+universe = vanilla
+log = checkpoint.log
+
+executable = run.sh
+
+checkpoint_exit_code = 85 
```

Comment:
add a comment about what this does? something like: 
`# indicate which exit code should trigger the job to continue running after checkpointing`